app/models/post.rb

```diff
@@ -1,3 +1,10 @@
 class Post < ApplicationRecord
-  belongs_to :author_id
+  belongs_to :author, class_name: :User, foreign_key: :author_id
+  has_many :comments, foreign_key: :post_id
+  has_many :likes, foreign_key: :post_id
+  after_save :update_posts_counter
+
```

Comment:
[REQUIRED] It is a requirement for post model to have a method that returns the 5 most recent comments for a given post.